packages/gnarly-core/src/ReducerRunner.ts

```diff
@@ -64,34 +63,46 @@ class ReducerRunner {
           // we're resuming, so replay from store if possible
           try {
             const latestTransaction = await globalState.store.getLatestTransaction(this.reducer.config.key)
```

Comment:
does getLatestTransaction include historicalblocks transaction or only other reducers tables?

Reply:
it doesn't; this gets the latest Ourbit transaction (which isn't a blockchain transaction, but I didn't have a better name for it :( )